src/node.cc

```diff
@@ -90,6 +90,7 @@
 
 #if defined(NODE_HAVE_I18N_SUPPORT)
 #include <unicode/uvernum.h>
+#include <unicode/putil.h>
```

Comment:
```suggestion
#include <unicode/utypes.h>
```
(I think that's the right header when you only need `U_ICUDATA_NAME`.)